TDAT: protected void tear down ( ) throws exception {  <NL> super . tear down ( ) ;  <NL>  <NL> log . trace ( " resetting the datastore to original state " ) ;  <NL>  <NL> connection connection = this . conn factory . get connection ( ) ;  <NL> statement statement = connection . create statement ( ) ;  <NL> statement . execute update (  <NL>  " update users set email _ id = null where id > 30 " ) ;  <NL> connection . commit ( ) ;  <NL> statement . close ( ) ;  <NL> connection . close ( ) ;  <NL>  }  COM: <s> in addition to destroying the remote cache instance it also initializes </s>